benchmarks/dynamo/common.py

```diff
@@ -942,10 +943,9 @@ def setup_amp(self):
             self.autocast = torch.cuda.amp.autocast
 
     def init_optimizer(self, device, params):
-        param_list = list(params)
-        if device == "cuda" and len(param_list) != 0:
+        if device == "cuda":
             # capturable is only supported on cuda at the moment
-            self.optimizer = torch.optim.Adam(param_list, capturable=True)
```

Comment:
can you remove `capturable=True` to simulate the real world where people aren't expected to set this?

Reply:
cc @mlazos Do you expect issues here? I am not aware of the full context.

Reply:
I was planning on having Dynamo patch this, and I was going to remove this after the patch was finished. I haven't had time to find a good way to patch it that is intuitive though. If we disable this there will be graph breaks in the optimizer due to `item()` calls. See the context here: https://github.com/pytorch/torchdynamo/issues/1083

Reply:
@soumith is it ok if we leave it as it is for now? And @mlazos can remove this later.